common/config/seq.go

```diff
@@ -28,6 +28,10 @@ func (j *Seq) ModifyConfig(dest ExporterConfigurer, currentConfig *Config) ([]st
 	if err != nil {
 		return nil, err
 	}
+	err = urlHostShouldNotContainPath(endpoint)
```

Comment:
We already execute url parsing above in `parseOtlpHttpEndpoint`.
What do you think to add this check into the existing function which we call anyway in all cases